createFolders.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+DIRECTORY="$(dirname "$(readlink "$0")")"
+
+if [[ $# -lt 1 ]]; then
+  echo "Usage: bash createFolders.sh <programming language>" >&2
+  exit 1
+fi
+
+programmingLanguage=$1
+mkdir "${DIRECTORY}/${programmingLanguage}"
```

Comment:
you should be able to drop this now since rsync will create it. if it doesn't, it might be because i forgot to add a trailing `/` to the destination directory ^^;